sdk/confidentialledger/azure-resourcemanager-confidentialledger/src/test/java/com/azure/resourcemanager/confidentialledger/ConfidentialLedgerManagementTest.java

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.confidentialledger;
+
+import com.azure.resourcemanager.confidentialledger.models.*;
```

Comment:
I am not sure why checkstyle does not report. But Java dev discourage the use of "import *". Would you break it done to the list of "import" of exact class?

Reply:
Sure, yes